src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp

```diff
@@ -139,5 +140,60 @@ class SubgraphBaseStaticTest : public ov::test::SubgraphBaseTest {
         ov::test::SubgraphBaseTest::run();
     }
 };
+
+inline std::string setToString(const std::unordered_set<std::string> s) {
+    if (s.empty())
+        return {};
+
+    std::string result;
+    result.append("{");
+    for (const auto& str : s) {
+        result.append(str);
+        result.append(",");
+    }
+    result.append("}");
+
+    return result;
+}
+
+inline void CheckNumberOfNodesWithTypeImpl(std::shared_ptr<const ov::Model> function,
+                                           const std::unordered_set<std::string>& nodeTypes,
+                                           size_t expectedCount) {
+    ASSERT_NE(nullptr, function);
+    size_t actualNodeCount = 0;
+    for (const auto& node : function->get_ops()) {
+        const auto& rtInfo = node->get_rt_info();
+        auto getExecValue = [&rtInfo](const std::string& paramName) -> std::string {
+            auto it = rtInfo.find(paramName);
+            OPENVINO_ASSERT(rtInfo.end() != it);
+            return it->second.as<std::string>();
```

Comment:
Detail, I think this `as` returns reference. Set lambda return type to same type as this function returns. Make new string is not required as it is used only to check if exists in `nodeTypes`.

Reply:
Yes, your idea is reasonable. But CheckNumberOfNodesWithType is called by 88 test cases, if I change it, I will have to change all called files. Can we track it by new ticket? @praasz